src/pages/index.js

```diff
@@ -9,33 +9,13 @@ import { handleClosePopupWithOutsideClick } from "../utils/utils.js";
 import Popup from "../components/Popup.js";
```

Comment:
```js

    this._popupElementCloseBtn.addEventListener("click", () => {
      this.close();
    });
```

Please, delete line 34 from `Popup`. You handle the close icon in line 28 already